src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -200,14 +204,27 @@ class LocalBackend extends Backend with LazyLogging {
   private def stageWdlFile(wdlFile: WdlFile, hostInputsPath: Path): WdlFile = {
     val originalPath = Paths.get(wdlFile.value)
     val executionPath = hostInputsPath.resolve(originalPath.getFileName.toString)
+
+    if (tryToHardlinkInputs) {
+      try {
+        Files.createLink(executionPath, originalPath)
```

Comment:
I'm guessing as the container wouldn't be able to navigate the host-relative path.